### ISSUE TITLE
After Updating a Test Case to JAMA, in the 'Upload Results' information section, it is shown the number of PASS and FAILED

### ISSUE THREAD

**Issue by JordiCela** [systelab/allure-reporter#103]
**Is your feature request related to a problem? Please describe.**
After Updating a Test Case to JAMA, in the 'Upload Results' information section, it is shown the number of PASS and FAILED

**Describe the solution you'd like**
After Uploading a Test Case to JAMA, in the 'Upload Results' information section, it must NOT be shown the number of PASS and FAILED

**Additional context**
![Image](https://user-images.githubusercontent.com/41988810/203117856-b901a72f-57b9-400d-a029-95b4dd0bffd6.jpg)

**Comment by danipiques** [comment]
Do you refer that the number of Pass and Fail only should be displayed with the option "Report"? And not with the option "Update Test Case"?

**Comment by JordiCela** [comment]
Yes, that's it.

**Comment by ManelMoreno** [comment]
YES, when updating to JAMA the test case definition/steps (NOT test results), PASS or FAIL does not apply.

**Comment by ManelMoreno** [comment]
This is a low priority bug,  affecting more to usability (confuse)

**Comment by ManelMoreno** [comment]
Evaluated and fix done in a PR.

**Comment by JordiCela** [comment]
**Environment:**
 OS: Windows 10 Pro 64 bits
 Browser: Google Chrome v.135
 Version: Allure Reporter v.4.0.0-beta


**Steps to reproduce the behavior:**

1. Go to Allure Reporter page
2. Drag the following 2 XML files (     ):
3. Login
4. Go to the 'Report to Jama' dialog
5. Select the 'Seed' project
6. Press the 'Update Test Cases' button

**Expected behavior:**
- The steps of the test cases are uploaded and updated in JAMA.
- After Uploading a Test Case to JAMA, in the 'Upload Results' information section, it is NOT shown the number of PASS and FAILED

**Results: PASSED**
**Evidences:**
